src/components/NavBar.scss

```diff
@@ -1,35 +1,60 @@
-.Nav {
-	background-color: var(--color-bg);
-	border-top: 1px solid var(--color-border);
-	bottom: 0;
-	display: flex;
-	flex-direction: row;
-	padding-bottom: max(env(safe-area-inset-bottom), 1rem);
-	padding-top: 1rem;
-	place-content: center;
-	position: fixed;
-	width: 100%;
-
-	&__container {
-		display: flex;
-		flex-direction: row;
-		justify-content: space-evenly;
-		width: min(72ch, 100%);
+@import "../styles/variables.scss";
+
+.navbar {
+	.navbar-brand {
+		font-size: 3.5rem !important;
+
+		text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.4);
+
+		@media (max-width: 430px) {
+			font-size: 3rem !important;
+		}
+		@media (max-width: 375px) {
+			font-size: 2.5rem !important;
+		}
 	}
 
-	&__link {
-		--color-text: var(--color-accent);
-		color: var(--color-text);
-		font-size: 1.4em;
-		flex: 0 1 auto;
+	.nav-link {
+		color: $primary-beige;
+		font-size: 1.5rem;
 		line-height: 1;
-		padding: 0.8rem;
-		text-align: center;
-		text-underline-offset: 0.1em;
 
 		&.active {
-			text-decoration-thickness: 0.22em;
-			text-underline-offset: 0.1em;
+			text-decoration: underline;
+			color: $primary-beige !important;
+			text-underline-offset: 0.3em;
+		}
+
+		&:hover {
+			color: $secondary-blue;
 		}
 	}
+
+	.navbar .navbar-toggler {
+		background-color: $primary-beige;
+	}
+}
+
+.navbar button {
+	background-color: $secondary-blue;
+	color: $primary-beige;
+	font-weight: bold;
+	border: none;
+
+	&:hover {
+		background-color: $primary-beige;
+		color: $secondary-blue;
+	}
+
```

Comment:
I noticed this interesting side effect  to the navbar button between 431 px and 768 px. it's only important if we want to target between those screen sizes. When between those screen widths, the navbar toggle is secondary blue and its difficult to see the hamburger to know its the menu button. I was playing with chromes responsive feature, and all the phones listed dont have this side effect so this might not be a issue we need to focus on. An observation! 
https://github.com/user-attachments/assets/aa9c995f-ae03-44c9-a2a1-6d6c7d8b95b6

Reply:
Good catch! I'll have to look at that maybe a different media query can be used 🤔